var/spack/repos/builtin/packages/openssl/package.py

```diff
@@ -23,13 +23,18 @@ class Openssl(Package):
     list_url = "https://www.openssl.org/source/old/"
     list_depth = 1
 
+    # Note: Version 1.1.1 is the current long-term support version that will
+    # remain supported until September 2023.
+    version('1.1.1',  '2836875a0f89c03d0fdf483941512613a50cfb421d6fd94b9f41d7279d586a3d')
+    version('1.1.0i', 'ebbfc844a8c8cc0ea5dc10b86c9ce97f401837f3fa08c17b2cdadc118253cf99')
     version('1.1.0g', 'ba5f1b8b835b88cadbce9b35ed9531a6')
     version('1.1.0e', '51c42d152122e474754aea96f66928c6')
     version('1.1.0d', '711ce3cd5f53a99c0e12a7d5804f0f63')
     version('1.1.0c', '601e8191f72b18192a937ecf1a800f3f')
-    # Note: Version 1.0.2 is the "long-term support" version that will
-    # remain supported until 2019.
-    version('1.0.2o', '44279b8557c3247cbe324e2322ecd114', preferred=True)
+    # Note: Version 1.0.2 is the previous long-term support version that will
+    # remain supported until December 2019.
+    version('1.0.2p', '50a98e07b1a89eb8f6a99477f262df71c6fa7bef77df4dc83025a2845c827d00', preferred=True)
```

Comment:
shouldn't we then default to 1.1.1?

Reply:
I would first like to check whether all dependents still build/work with 1.1.1. As far as I remember, there have been some problems in Ubuntu when they switched to 1.1.1.

Reply:
We should probably have a policy for how to deal with this...

Reply:
FYI: I am currently running builds for all dependents and will submit a separate PR for switching the preferred version.